### PR TITLE
Clean-up exits (no Ctrl-C/Alt-Q)

### PR DESCRIPTION
**Pull request by zcsahok**
As discussed in #247

### CODE REVIEW

File: share/help.txt [hunk 1]
```diff
@@ -8,18 +8,18 @@ Alt-T     Tune                          Alt-N   Note in log
 Alt-V     Band down (CW Speed?)         Alt-P   Turn PTT on/off
 Alt-H     Show help.txt                 Alt-W   CW Weight
 Alt-I     Show talk messages            Alt-J   See QRG's
-Alt-K     CW from Keyboard              Alt-X   Exit program (Alt-q)
-Alt-M     Multi display (Alt-c)         Alt-S   Score window (Alt-r)
-Alt-,     Show bandmap                  Alt-J   Show frequency
+Alt-K     CW from Keyboard              Alt-X   Exit program
+Alt-M     Multi display (Alt-C)         Alt-S   Score window (Alt-R)
 Alt-Z     Show worked zones             Alt-G   Grab call (min. 2 letters from bandmap)
+Alt-,     Show bandmap
```
Comment (dl1jbe) [inline share/help.txt:14]: Why did you drop `Alt-J`. It still works and is useful in multi op operation.

**Review by dl1jbe — COMMENTED** [review]
The rest looks good. Go ahead and merge it in.

File: share/help.txt (hunk 1 above)
Comment (zcsahok) [reply share/help.txt:14]: It's still there, just dropped the second one and moved Alt-, to the end of the list.
```
   Alt-J   See QRG's
   Alt-J   Show frequency
```

Comment (dl1jbe) [reply share/help.txt:14]: Silly me. I see, it was there two times. You dropped only the second one.

**Pull request merged**